src/db_window.rs

```diff
@@ -194,54 +193,54 @@ mod test {
         t_receiver.join().expect("join");
         t_responder.join().expect("join");
     }
-
-    #[test]
-    pub fn test_send_to_retransmit_stage() {
-        let leader = Keypair::new().pubkey();
-        let nonleader = Keypair::new().pubkey();
-        let mut leader_scheduler = LeaderScheduler::default();
-        leader_scheduler.set_leader_schedule(vec![leader]);
-        let leader_scheduler = Arc::new(RwLock::new(leader_scheduler));
-        let blob = SharedBlob::default();
-
-        let (blob_sender, blob_receiver) = channel();
-
-        // Expect all blobs to be sent to retransmit_stage
-        blob.write().unwrap().forward(false);
-        retransmit_blobs(
-            &vec![blob.clone()],
-            &leader_scheduler,
-            &blob_sender,
-            &nonleader,
-        )
-        .expect("Expect successful retransmit");
-        let _ = blob_receiver
-            .try_recv()
-            .expect("Expect input blob to be retransmitted");
-
-        blob.write().unwrap().forward(true);
-        retransmit_blobs(
-            &vec![blob.clone()],
-            &leader_scheduler,
-            &blob_sender,
-            &nonleader,
-        )
-        .expect("Expect successful retransmit");
-        let output_blob = blob_receiver
-            .try_recv()
-            .expect("Expect input blob to be retransmitted");
-
-        // retransmit_blobs shouldn't be modifying the blob. That is retransmit stage's job now
-        assert_eq!(*output_blob[0].read().unwrap(), *blob.read().unwrap());
-
-        // Expect blob from leader while currently leader to not be retransmitted
-        // Even when forward is set
-        blob.write().unwrap().forward(true);
-        retransmit_blobs(&vec![blob], &leader_scheduler, &blob_sender, &leader)
+    /*
```

Comment:
Planning to ship it like this? Usually we `#[ignore]` or delete. Does this test still have value?

Reply:
I was planning to file an issue. I can uncomment and make it compile. But that'll still leave it broken.

Reply:
You can leave it. Let's get it merged.  With the `#[ignore]` flag, it doesn't even need to compile,  sadly. It's the same as commenting it out. Rust needs XFAIL.

Reply:
#2966